alpha/helmfile.yaml

```diff
@@ -2,12 +2,20 @@
 repositories:
   - name: datarepo-helm
     url: https://broadinstitute.github.io/datarepo-helm
+  - name: descheduler
+    url: https://kubernetes-sigs.github.io/descheduler
 
 helmDefaults:
   createNamespace: true
 
 # helm releases to be deployed
 releases:
+  - name: descheduler   # release name
```

Comment:
Nice!  Weren't we just talking about this a few weeks back?

Reply:
Yeah its been working in dev for a month now